test/parallel/test-https-client-get-url.js

```diff
@@ -31,7 +29,12 @@ var server = https.createServer(options, function(req, res) {
 });
 
 server.listen(common.PORT, function() {
-  https.get('https://127.0.0.1:' + common.PORT + '/foo?bar');
+  https.get({
+    host: '127.0.0.1',
+    port: common.PORT,
+    path: '/foo?bar',
+    rejectUnauthorized: false
+  });
 });
```

Comment:
> Can node tests assume internet access?

Tests that run regularly do not go out to the internet (as far as I'm aware). The tests that do go out to the internet end up in `test/internet` where they are run infrequently. (Regular CI runs, for example, do not exercise those tests. It is possible that they are running during the release process or at other times. I don't know. Maybe someone who does know can chime in.)